ansible_base/lib/dynamic_config/dynamic_settings.py

```diff
@@ -26,6 +26,21 @@
     REST_FRAMEWORK = {}
 
 
+# Explanation - these are the filters for views provided by DAB like /authenticators/
+# we want them to be enabled by default _even if_ the rest_filters app is not used
+# so that clients have consistency, but if an app wants to turn them off, they can
+# these will be combined with the actual DRF defaults in our base view
+try:
+    ANSIBLE_BASE_CUSTOM_VIEW_FILTERS
```

Comment:
How often will we "pay the price" for the loop (which is probablly insignificant to begin with). Just thinking should we merge this somehow with the if condition down on line 51 `if 'ansible_base.rest_filters' in INSTALLED_APPS:` that we set ANSIBLE_BASE_CUSTOM_VIEW_FILTERS to `None` or `()` so that we avoid the loop altogether on services that use the rest_filters (which would automatically add the class). Or we could wrap the for loop in the if. I will still approve as I don't think this is blocking.

Reply:
Valid comment from a DRY perspective (not really a performance perspective). I'll push a commit in a bit. This will "avoid the loop" on services that use rest_filters, like you suggest. It also avoids catching `NameError`, which is generally weird. It'll set `ANSIBLE_BASE_CUSTOM_VIEW_FILTERS` in the `else:` of `if 'ansible_base.rest_filters' in INSTALLED_APPS:` which I agree is the intuitive expectation. Instead of `NameError`, this will use `getattr(settings, 'ANSIBLE_BASE_CUSTOM_VIEW_FILTERS', ())`